charts/ziti-controller/templates/ca-router-ctrl-identity.yaml

```diff
@@ -114,9 +114,8 @@ spec:
     - {{ include "ziti-controller.fullname" . }}-ctrl
     - {{ include "ziti-controller.fullname" . }}-ctrl.{{ .Release.Namespace }}
     - {{ include "ziti-controller.fullname" . }}-ctrl.{{ .Release.Namespace }}.svc
-    - {{ include "ziti-controller.fullname" . }}-ctrl.{{ .Release.Namespace }}.svc.cluster
-    - {{ include "ziti-controller.fullname" . }}-ctrl.{{ .Release.Namespace }}.svc.cluster.local
-    - {{ .Values.ctrlPlane.advertisedHost | default (printf "%s-ctrl.%s.svc.cluster.local" .Release.Name .Release.Namespace) }}
+    - {{ include "ziti-controller.fullname" . }}-ctrl.{{ .Release.Namespace }}.svc.{{ .Values.ca.clusterDomain }}
```

Comment:
Ah, cool. Nice catch that `cluster.local` is a DNS atom representing the top level zone of cluster DNS. 

Yes, we can make this configurable. We just need a default value for `ca.clusterDomain=cluster.local` in the values file.

Reply:
done! somehow the values file didn't get pushed up the first time.